src/hotspot/share/runtime/arguments.cpp

```diff
@@ -4205,6 +4205,7 @@ jint Arguments::apply_ergo() {
   }
 #endif // COMPILER2
 
+
```

Comment:
Unnecessary changes, that bring one more file to diff list. 
Please fix.

Reply:
Fixed.